portal/portal-sprint-102.md

```diff
@@ -0,0 +1,58 @@
+---
+slug: /release-notes/portal/sprint-102
+title: Sprint 102 (June 14, 2021)
+---
+
+# fiskaltrust.Portal - Sprint 102
+_June 14, 2021_
+
+## Features
+
+### Middleware Configuration
+- [Visual improvements to Middleware component dialogues](#visual-improvements-to-middleware-component-dialogues)
+- [Preview of Public Dashboard for Uptime of our Services](#preview-of-public-dashboard-for-uptime-of-our-services)
+
+### Support
+
+- [KBArticles improvements](#kbarticles-improvements)
+
+## Middleware Configuration
+
+### Visual improvements to Middleware component dialogues
+
+Some minor improvements of the SignaturCreationUnit DE TableView(DE)  in Portal have been implemented. We invested some time in improving the way our tables are displayed. All previous functionalities are kept, but the consistency of all operations, as well as the performance of the page in different resolutions, has also been improved.
+
+![scu](images/sprint-102/scu.png)
+
+### Preview of Public Dashboard for Uptime of our Services
+
+A Preview of the Public Dashboard for Uptime of our Services is available, with options such as detailed view on response time, events view and overall uptime metrics.
```

Comment:
How can I get to this preview?

Reply:
I am not sure I know what you mean :(

Reply:
You describe here that we have this cool new dashboard showing the service availability, but nowhere mention how or where to access this feature. If someone wants to test or take a look at it, they simply can't find it. (Including me 😅 )

Reply:
aha, very good point 😄  I don't know though where to find it myself so I'd need your help here 😇